app/Plugin/CoreApi/Config/routes.php

```diff
@@ -50,7 +64,13 @@
   )
 );
 
+// XXX Scoped identifiers are not parsed properly because they are perceived as file extensions
```

Comment:
Ah yes, definitely a problem. But why is this still flagged `XXX`?

Reply:
Good point. I will remove this.